config/webpack.config.js

```diff
@@ -582,5 +583,12 @@ module.exports = async (env = {}) => {
     );
   }
 
+  if (buildOptions.lightweightWatch) {
+    baseConfig.watchOptions = {
+      ignored: '**/node_modules',
+      poll: true,
+    };
+  }
```

Comment:
I didn't completely vet `ignored`. This PR is not necessarily claiming to be more than a conversation starter though.